slices/libtasn1-6.yaml

```diff
@@ -0,0 +1,8 @@
+package: libtasn1-6
+
+slices:
+  libs:
+    essential:
+      - libc6_libs   
+    contents:
+      /lib/*-linux-*/libtasn1.so.6*:  {arch: amd64}
```

Comment:
Why are we extracting it only for `arch: amd64`?

Reply:
Last time I checked the https://packages.ubuntu.com/noble/libs/libtasn1-6 the packages was showing only for amd64.

I will update all the packages for 24.04 and 23.10

Reply:
Oh! Packages for 24.04 are changing a lot recently, so maybe that's the case.